role-make-version-changelog.sh

```diff
@@ -55,7 +55,12 @@ get_main_branch() {
     return 1
 }
 
-git fetch --all
+commit_to_file() {
+    git log --oneline --no-merges --reverse --pretty=format:"- %s%n%n%w(80,2,2)%b%n%n" -1 "$1" | \
+        awk 'NF > 0 {blank=0} NF == 0 {blank++} blank < 2' >> "$2"
```

Comment:
```suggestion
        awk 'NF > 0 {blank=0} NF == 0 {blank++} blank < 2'
```
I prefer to write shell functions like this to work in a wide variety of contexts - instead of passing in the argument which is the file to write to, just output to stdout and let the caller redirect, pipe, etc. - same with input files although not applicable to this function